homework_1/zad2/taxLibrary/taxLibrary_test.go

```diff
@@ -0,0 +1,29 @@
+package taxLibrary_test
+
+import (
+	"code-cadets-2021/homework_1/zad2/taxLibrary"
+	"testing"
+)
+
+func TestCalculateTax(t *testing.T) {
+	for _, tc := range getTestCases() {
```

Comment:
c/p from a comment I left on another homework:

I know abbreviations are quite "idiomatic" in Go, but, in the day of modern IDEs that don't require you to type the entire variable name + no technical restrictions on the length of the variable names as there were 50 years ago - I personally prefer (and encourage/enforce) using more verbose but more informative names - I leave this up to you to decide what to do and how, what you wrote is also ok